pkg/tfgen/installation_docs.go

```diff
@@ -128,17 +138,25 @@ func stripSchemaGeneratedByTFPluginDocs(content []byte) []byte {
 	return content
 }
 
-func applyEditRules(contentBytes []byte, docFile string, g *Generator) ([]byte, error) {
+func applyProviderEditRules(contentBytes []byte, docFile string, g *Generator) ([]byte, error) {
 	// Obtain edit rules passed by the provider
 	edits := g.editRules
+	contentBytes, err := edits.apply(docFile, contentBytes)
+	if err != nil {
+		return nil, err
+	}
+	return contentBytes, nil
+}
+func applyInstallationEditRules(contentBytes []byte, docFile string) ([]byte, error) {
+	var edits editRules
 	// Additional edit rules for installation files
 	edits = append(edits,
```

Comment:
If we have a fixed set of edit rules, we don't need to `append`.

```suggestion
	edits := editRules{
```